src/components/Cards/ForgottenPassword.vue

```diff
@@ -14,14 +17,21 @@
         required
         v-model="email"
         :disabled="inFlight"
-        :error-messages="error"
         />
       </v-form>
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn @click="reset" color="primary" :disabled="inFlight">{{buttonText}}</v-btn>
+      </v-card-actions>
+      <v-alert class="mt-8 mr-2 ml-2" outlined type="error" border="left" v-if="error">
+        {{error}}
+      </v-alert>
+      <v-alert class="mt-8 mr-2 ml-2" outlined type="success" border="left" v-if="success">
+        An email has been sent to <b>{{email}}</b>
```

Comment:
This message is shown if I enter anything or nothing. Shouldn't there be at least a check that what is entered wasn't empty and possibly an email?

Reply:
we also talked about it as I remember, but the ticket doesn't mention it and also there's no design to do so

Reply:
This could be a follow-up ticket I guess

Reply:
resolved to check that it is not empty with the rules (and validation on submit) on the `v-text-field`